index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./home.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <title>warzone_world_series</title>
+</head>
+
+<body>
+  <div class="mobile-menu" id="mobile-menu">
+    <div class="mobile-menu-header">
+      <button type="button" data-close-button class="close-button">&times;</button>
+    </div>
+    <nav>
+      <a href="./index.html" data-close-button class="navigate">
+        Home
+      </a>
+      <a href="./about.html" data-close-button class="navigate">
+        About
+      </a>
+      <a href="./index.html#program" data-close-button class="navigate">
+        Program
+      </a>
+      <a href="#" data-close-button class="navigate">
+        Jobs
+      </a>
+      <a href="./about.html#sponsor" data-close-button class="navigate">
+        Sponsor
+      </a>
+    </nav>
+  </div>
+  <header>
+
```

Comment:
- [ ] Please add the top short bar section with social media links, logout, and language links as you can see in the next image.

![Screen Shot 2021-11-17 at 19 28 30](https://user-images.githubusercontent.com/67211919/142334709-c1ce99a9-f764-44bf-8179-4761ba1174d6.png)

Reply:
Oh, I did not see that. Thank you for that observation. I have added the short nav bar